pss/pss_test.go

```diff
@@ -48,28 +50,95 @@ func TestTrojanChunkRetrieval(t *testing.T) {
 	payload := []byte("RECOVERY CHUNK")
 	topic := trojan.NewTopic("RECOVERY")
 
-	var ch chunk.Chunk
+	var monitor *Monitor
 
 	// call Send to store trojan chunk in localstore
-	if ch, err = pss.Send(ctx, testTargets, topic, payload); err != nil {
+	if monitor, err = pss.Send(ctx, testTargets, topic, payload); err != nil {
 		t.Fatal(err)
 	}
 
 	// verify store, that trojan chunk has been stored correctly
 	var storedChunk chunk.Chunk
-	if storedChunk, err = localStore.Get(ctx, chunk.ModeGetRequest, ch.Address()); err != nil {
+	if storedChunk, err = localStore.Get(ctx, chunk.ModeGetRequest, monitor.chunk.Address()); err != nil {
 		t.Fatal(err)
 	}
+	storedChunk = storedChunk.WithTagID(monitor.chunk.TagID())
 
-	if !reflect.DeepEqual(ch, storedChunk) {
+	if !reflect.DeepEqual(monitor.chunk, storedChunk) {
 		t.Fatalf("store chunk does not match sent chunk")
 	}
 
 	// check if pinning makes a difference
 
 }
 
-func newMockLocalStore(t *testing.T) *localstore.DB {
+// TestPssMonitor creates a trojan chunk
+// mocks the localstore
+// calls pss.Send method
+// updates the tag state (Stored/Sent/Synced)
+// waits for the monitor to notify the changed state
+func TestPssMonitor(t *testing.T) {
+	var err error
+	ctx := context.TODO()
+	tags := chunk.NewTags()
+
+	localStore := newMockLocalStore(t, tags)
+
+	testTargets := [][]byte{
```

Comment:
you make this look like it matters, keep params simple when it is irrelevant

Reply:
I changed it to `targets` as other parts of the test

Reply:
not sure, but i think @zelig might have meant that in this case, it doesn't really add to the test that we have 6 bytes here.

it's also going to take more work to call `Wrap` on such data, and the `trojan` package already takes care of properly testing that feature.